test/model/tools/model_test/main.cpp

```diff
@@ -285,6 +431,7 @@ main(int argc, char *argv[])
     bool preserve = false;
     bool print_only = false;
     const char *progname = argv[0];
+    bool reduce = false;
```

Comment:
We kinda spoke about this offline - I think `true` would be a nicer default. I suspect the first thing someone is going to try when seeing a verifier failure is to reduce it, so we may as well have a go first.

Reply:
Good idea. Fixed in [99d0adb](https://github.com/wiredtiger/wiredtiger/pull/10318/commits/99d0adb57880cb11dd452ea30ba81ee3649e1e9b).